src/Entity/Balance.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the DigitalOceanV2 library.
+ *
+ * (c) Antoine Corcy <contact@sbin.dk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace DigitalOceanV2\Entity;
+
+/**
+ * @author Simon Bennett <simon@snapshooter.io>
+ */
+final class Balance extends AbstractEntity
+{
+    /**
+     * @var string
+     */
+    public $monthToDateBalance;
+
+    /**
+     * @var string
```

Comment:
is this a string? what's an example value?

Reply:
I based off the digitalocean api definition of attribute with money been a string.

This is inline with the rest of the project too 

https://github.com/DigitalOceanPHP/Client/blob/d000d351d931902d4bb26e2836a3bf52cba28c86/src/Entity/Size.php#L55